unit-tests/test-fw-update.py

```diff
@@ -242,6 +242,20 @@ def find_image_or_exit( product_name, fw_version_regex = r'(\d+\.){3}(\d+)' ):
 time.sleep(3) # MIPI devices do not re-enumerate so we need to give them some time to restart
 devices.query( monitor_changes = False )
 sn_list = devices.all()
+sn_list_size = len(sn_list)
+if sn_list_size == 0:
+    retries = 10
```

Comment:
30 seconds timeout is too long IMO.
Maybe we can do 5x2 sec?

Reply:
It is 3 seconds at each iteration, and it usually succeeds at the third iteration.
30 seconds is only the worst case - I agree it should not occur...
So you mean 5 iterations of 2 seconds each - OK.